packages/notifier/src/index.js

```diff
@@ -1,5 +1,7 @@
 // @jessie-check
 
+import '@agoric/internal/exported.js';
```

Comment:
is it worth an XXX or something on these? or UNTIL #6512 ?

Reply:
there are tons of these but I tried to comment a bunch of them